tika-parsers/src/main/java/org/apache/tika/parser/crypto/TSDParser.java

```diff
@@ -0,0 +1,361 @@
+		
+        try {
+             
+        	 //log.info("TSDParser --> buildMetas START!");
+             CMSTimeStampedDataParser cmsTimeStampedData = new CMSTimeStampedDataParser(stream);
+             
+			 TimeStampToken[] tokens = cmsTimeStampedData.getTimeStampTokens();
+			 
+			 for (int i=0; i < tokens.length; i++) {
+				 
+				 TSDMetas tsdMetas = new TSDMetas(true,
+						 						  tokens[i].getTimeStampInfo().getGenTime(),
+						 						  tokens[i].getTimeStampInfo().getPolicy().getId(),
+						 						  tokens[i].getTimeStampInfo().getSerialNumber(),
+						 						  tokens[i].getTimeStampInfo().getTsa(),
+						 						  tokens[i].getTimeStampInfo().getHashAlgorithm().getAlgorithm().getId());
+				 
+				 //log.info("TSDParser --> buildMetas genTime: " + tsdMetas.getEmitDateStr());
+				 //log.info("TSDParser --> buildMetas policyId: " + tsdMetas.getPolicyId());
+				 //log.info("TSDParser --> buildMetas serialNumber: " + tsdMetas.getSerialNumberFormatted());
+				 //log.info("TSDParser --> buildMetas tsa: " + tsdMetas.getTSAstr());
+				 //log.info("TSDParser --> buildMetas algorithm: " +  tsdMetas.getAlgorithmName());
+				 
+				 //log.info("TSDParser --> tsdMetas instance built: " + tsdMetas);
+				 tsdMetasList.add(tsdMetas);
+			 }
+			 
+             //log.info("TSDParser --> tsdMetas list: " + tsdMetasList);
+             //log.info("TSDParser --> buildMetas END!");
+             
+        } catch (Exception ex) {
+             //log.error("TSDParser --> buildMetas Error: ", ex);
+             tsdMetasList = new ArrayList<TSDMetas>();
+        }
+        
+        return tsdMetasList;
+	}
+	
+	private class TSDMetas {
+		
+		private final String DATE_FORMAT = "dd/MM/yyyy HH:mm:ss";
+		
+		private Boolean parseBuilt = false;
+		private Date emitDate = new Date();
+		private String policyId = "";
+		private BigInteger serialNumber = null;
+		private GeneralName TSA = null;
+		private String algorithm = "";
+		private Date parsedDate = new Date();
+		
+		public TSDMetas() {
+			super();
+		}
+		
+		public TSDMetas(Boolean parseBuilt, Date emitDate, String policyId,
+				BigInteger serialNumber, GeneralName tSA, String algorithm) {
+			super();
+			this.parseBuilt = parseBuilt;
+			this.emitDate = emitDate;
+			this.policyId = policyId;
+			this.serialNumber = serialNumber;
+			this.TSA = tSA;
+			this.algorithm = algorithm;
+		}
+		
+		public Boolean getParseBuilt() {
+			return parseBuilt;
+		}
+		
+		public String getParseBuiltStr() {
+			return String.valueOf(this.getParseBuilt()!=null ? 
+								  this.getParseBuilt(): false);
+		}
+		
+		public void setParseBuilt(Boolean parseBuilt) {
+			this.parseBuilt = parseBuilt;
+		}
+		
+		public Date getEmitDate() {
+			return emitDate;
+		}
+		
+		public void setEmitDate(Date emitDate) {
+			this.emitDate = emitDate;
+		}
+		
+		public String getEmitDateStr() {
+			SimpleDateFormat sdf = new SimpleDateFormat(this.DATE_FORMAT, Locale.ITALY);
```

Comment:
Why this format and `Locale.ITALY` is used? Maybe `Locale.ROOT` to make it more or less locale-agnostic? Is there some fixed date format in TSD like RFC822/2822/5322 one?